src/sentry/static/sentry/app/views/issueList/congratsRobots.jsx

```diff
@@ -1,61 +0,0 @@
-import React from 'react';
-import styled from '@emotion/styled';
-
-import {t} from 'app/locale';
-import space from 'app/styles/space';
-import video from 'app/../images/spot/congrats-robots.mp4';
-
-const Message = () => (
-  <React.Fragment>
-    <EmptyMessage>
-      {t("We couldn't find any issues that matched your filters.")}
-    </EmptyMessage>
-    <p>{t('Get out there and write some broken code!')}</p>
-  </React.Fragment>
-);
-
-const CongratsRobots = () => (
-  <CongratsRobotsWrapper>
-    <AnimatedScene>
-      <StyledVideo autoPlay loop>
-        <source src={video} type="video/mp4" />
-        {/* Show message if browser doesn't support video */}
-        <Message />
```

Comment:
Wouldn't this show a double `Message` since we have another one side of the Wrapper?